index.js

```diff
@@ -11,6 +11,7 @@ const HistoricAlert = require('./lib/historicAlert')
 const Location = require('./lib/location')
```

Comment:
Can you please have a scan through the `README` in this repo? It goes more in-depth into documenting the functions that our other repos. It probably needs updating to reflect your changes.

Reply:
Added descriptions for my additions. :)